app/assets/javascripts/analytics/init.js.erb

```diff
@@ -24,7 +24,5 @@
 
   if (global.ga) {
     var ga = global.ga;
-    ga('require', 'linker');
-    ga('linker:autoLink', ['planforbritain.gov.uk']);
   }
```

Comment:
Leaving a questing comment on my own PR: is this assignment still needed? I'm not so hot on JS scoping, and the ga variable is used in ecommerce.js and static-analysis.js: are those instances of ga this ga?